.github/scripts/build_triton_wheel.py

```diff
@@ -102,7 +99,7 @@ def build_triton(
         if release:
             ver, rev, patch = version.split(".")
             if build_rocm:
-                check_call(["git", "checkout", "release/2.2.x"], cwd=triton_basedir)
+                check_call(["git", "checkout", "release/2.3.x"], cwd=triton_basedir)
```

Comment:
@jataylo This can be further simplified by removing this `build_rocm` condition, since the default `else` condition will pick up `release/2.3.x` anyway

Reply:
Ah good catch, updating now.